Lastfm/Client/LastfmAPIClient.php

```diff
@@ -47,13 +48,26 @@ protected function call(array $params = array())
         $httpQuery = http_build_query($params);
         curl_setopt($this->cURL, CURLOPT_POSTFIELDS, $httpQuery);
         $cURLResponse = curl_exec($this->cURL);
+
+        // better fix for malformed returned <type>.search xml from API
```

Comment:
you could encapsulated this whole block in a private function - so it's potentialy easier to refactor later on